apps/src/templates/sectionProgressV2/LevelDataCell.jsx

```diff
@@ -16,6 +18,14 @@ import ProgressIcon from './ProgressIcon';
 import legendStyles from './progress-table-legend.module.scss';
 import styles from './progress-table-v2.module.scss';
 
+const levelClickedAmplitude = (sectionId, isAssessment) => () => {
+  console.log('levelClickedAmplitude', sectionId, isAssessment);
+  analyticsReporter.sendEvent(EVENTS.PROGRESS_V2_VIEW, {
```

Comment:
Should this be sending the event `EVENTS.PROGRESS_V2_VIEW_LESSON_DETAILS`? (but also see my comment above about naming)